docs/DeveloperGuide.md

```diff
@@ -1,18 +1,149 @@
-# Developer Guide
+# Developer Guide - Diet Manager
 
-## Design & Implementation
+## Content
+1. [Design](#design)
+2. [Implementation](#implementation)
+2. [Product Scope](#product-scope)
+3. [User Stories](#user-stories)
+4. [Non-Functional Requirements](#non-functional-requirements)
+5. [Glossary](#glossary)
+6. [Instructions for Manual Testing](#instructions-for-manual-testing)
+
+## Design
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Architecture
+
+![ArchitectureDiagram](/docs/images/Architecture.png)
+
+The Architecture Diagram given above explains the high-level design of the Diet Manager Application.
+
```

Comment:
The main component  `Main` is not elaborated at the start of the Architecture ? 🤔  It would be great to let the developers know ?